modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js

```diff
@@ -71,17 +71,19 @@ const Sidebar = ({children, onClose = noop, open = true}) => {
 		}
 	}, [delay, open]);
 
+	useEffect(() => {
+		const body = document.querySelector('body');
+
+		isOpen
```

Comment:
My preference would be to use the ternary operator for expression contexts (eg. `const x = a ? b : c` and so on). Here statements would be appropriate; eg:

```js
if (isOpen) {
    // ...
}
else {
    // ...
}
```

I know it's more lines, but it makes the distinction clearer between this (a procedural, imperative section) and other more pure, functional bits.

Reply:
roger that